### DISCUSSION TITLE
Error "getTranslations is not supported in Client Components" although used in async RSC

### DISCUSSION THREAD

**Discussion by riccardolardi** [amannn/next-intl#1330] [Q&A] [answered]
I'm calling `getTranslations` from within an async RSC that is being called from a server side node-cron function but I'm getting the error `getTranslations is not supported in Client Components` which is weird since it is not a client component... possibly related to https://github.com/amannn/next-intl/issues/1232 ?

**Comment by amannn** [comment, marked as answer]
You're trying to use `getTranslations` in a Node.js-based cron job that's running outside of Next.js, right? `getTranslations` only works in Next.js, in this case you might want to consider using the [core library](https://next-intl-docs.vercel.app/docs/environments/core-library).
